tests/lib/rules/enforce-no-external-url-test.js

```diff
@@ -29,11 +29,6 @@ ruleTester.run(
             },
             {
                 code:`
-<script src="foo.js" />
```

Comment:
This is removing a duplicate test (was copy paste error I guess) which ESLint9 detected and was failing